Cython/Utility/ObjectHandling.c

```diff
@@ -1289,6 +1289,15 @@ static CYTHON_INLINE PyObject *__Pyx__GetModuleGlobalName(PyObject *name)
     } else if (unlikely(PyErr_Occurred())) {
         return NULL;
     }
+#elif CYTHON_COMPILING_IN_LIMITED_API
+    PyObject *$module_cname = PyState_FindModule(&$pymoduledef_cname);
+    if (unlikely($module_cname == NULL)) {
```

Comment:
We tend to prefer `!variable` over `variable == NULL` in the C code, although not consistently. Just saying.

Reply:
Still TODO, will come back to this

Reply:
Done!

Reply:
Done